.github/workflows/patch-release.yml

```diff
@@ -0,0 +1,105 @@
+name: 'Patch Release'
+
+on:
+  workflow_dispatch:
+    inputs:
+      type:
+        description: 'The type of release to patch from.'
+        required: true
+        type: 'choice'
```

Comment:
Docs seem to have type unquoted, please unquote.

ref: https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#example-of-onworkflow_dispatchinputs